_components/button/guidance/implementation.md

```diff
@@ -1,4 +1,3 @@
-- The component preview and component code demonstrate how to use button elements. To use a button style on a link, add the `usa-button` class to your link.
-- To use a different style button on your link, add button variant classes in addition to `usa-button`.
-- Use the browser-native `disabled` attribute for any disabled button. Don't use `usa-button--disabled`, which is intended only for debugging and prototyping.
-- Always set the `type` attribute to define the purpose of the button. The type attribute can accept three values: `submit`, `button`, and `reset`. If no type attribute is defined, the button will behave as a `submit` button.
\ No newline at end of file
+- **Always set the type attribute.** Define the purpose and behavior of a button with the `type` attribute. The `type` attribute can accept three values: `submit`, `button`, and `reset`. If no type attribute is defined, the button will behave as a `submit` button.
+- **Don't use the `usa-button--disabled` class.** Use the browser-native `disabled` attribute for any disabled button. Don't use `usa-button--disabled`, which is intended only for debugging and prototyping.
```

Comment:
@mejiaj Great catch. What do you think about removing mention of `usa-button--disabled` entirely since we aren't showing it in the component code anymore? We could rewrite the section to focus on how to turn on disabled styles. Something like:

```suggestion
- **Use the `disabled` or `aria-disabled` attributes to turn on disabled button styles.**
```

Reply:
I think this is a great idea!

Reply:
Updated the content of this item in [54120b2](https://github.com/uswds/uswds-site/pull/2149/commits/54120b21ea5ec8e86cf7d7f58e2e9b4369dc2aa6)